### PR TITLE
feat(condo): DOMA-7056 added validation for roles OrganizationEmployee schema. Added 'Ticket' for 'UserRightsSet'

### PR DESCRIPTION
**Pull request by Alllex202**
<s>Problem: support cannot invite or create OrganizationEmployee for service user. 
In this case, this is necessary so that on behalf of service user, webhook can receive data from organizations connected to CC

(It used to be possible to create, but in one of the updates it was forbidden)

Now support can create employee, but only if specified user
</s>

Also: added validation for role. You can use role only from same organization

Upd: This problem can be solved using `UserRightsSet` and filtering `Ticket`'s by `relatedOrganization`. Therefore for `UserRightsSet` was added `Ticket` schema in config

### CODE REVIEW

File: apps/condo/domains/organization/schema/OrganizationEmployee.test.js [hunk 1]
```diff
@@ -56,17 +56,36 @@ describe('OrganizationEmployee', () => {
...
+        expect(obj.dv).toEqual(1)
+        expect(obj.sender).toEqual(attrs.sender)
+        expect(obj.v).toEqual(1)
+        expect(obj.createdBy).toEqual(expect.objectContaining({ id: support.user.id }))
+        expect(obj.updatedBy).toEqual(expect.objectContaining({ id: support.user.id }))
+        expect(obj.createdAt).toMatch(DATETIME_RE)
+        expect(obj.updatedAt).toMatch(DATETIME_RE)
+    })
+
+    test('support: cannot create OrganizationEmployee for one user in one organization twice', async () => {
```
Comment (crowncode-development) [inline apps/condo/domains/organization/schema/OrganizationEmployee.test.js:78]: create employee with role from another organization

File: apps/condo/domains/organization/access/OrganizationEmployee.js [hunk 2]
```diff
@@ -30,13 +32,44 @@ async function canReadOrganizationEmployees ({ authentication: { item: user } })
...
 }
 
-async function canManageOrganizationEmployees ({ authentication: { item: user }, operation, itemId }) {
+async function canManageOrganizationEmployees ({ authentication: { item: user }, operation, itemId, originalInput }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin) return true
+
+    if (user.isSupport) {
+        if (operation === 'create') {
```
Comment (pahaz) [inline apps/condo/domains/organization/access/OrganizationEmployee.js:41]:
```suggestion
    // NOTE: support can create an Employee with a role from the same organization!
    if (user.isSupport && operation === 'create') {
```

File: apps/condo/domains/organization/access/OrganizationEmployee.js [hunk 3]
```diff
@@ -30,13 +32,44 @@ async function canReadOrganizationEmployees ({ authentication: { item: user } })
...
+            }
+
+            const employeeRole = await getByCondition('OrganizationEmployeeRole', {
+                id: roleId,
+                organization: { id: organizationId },
+            })
+
+            return !!employeeRole
+        } else {
+            return true
```
Comment (pahaz) [inline apps/condo/domains/organization/access/OrganizationEmployee.js:67]: we also need to check role if update

Comment (Alllex202) [reply apps/condo/domains/organization/access/OrganizationEmployee.js:67]: done

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/user/utils/directAccess/config.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ const DIRECT_ACCESS_AVAILABLE_SCHEMAS = {
         'B2CAppAccessRight',
         'B2CAppBuild',
         'B2CAppProperty',
+        'Ticket',
```
Comment (SavelevMatthew) [inline apps/condo/domains/user/utils/directAccess/config.js:26]: Please add Domain-comment-separator and keep things alphabetically order.

P.S.: Right now it's looking clean, but once most of domains will be migrated it'll became hard to read (remember translations mess), so alphabetic domains + alphabetic inside 🙏

Comment (Alllex202) [reply apps/condo/domains/user/utils/directAccess/config.js]: done

**Review by SavelevMatthew — APPROVED** [review]

**Pull request merged**